package.json

```diff
@@ -39,7 +39,7 @@
   ],
   "scripts": {
     "watch": "parcel watch",
-    "build": "parcel build && yarn run css && tsc",
+    "build": "parcel build && yarn run css && copyfiles -f types/dropzone.d.ts dist",
```

Comment:
do we need copyfiles? wouldn't `cp` work?

Reply:
Not on windows

Reply:
The problem is that copyfiles brings with it a lot of dependencies (and seems unmaintained, too). I know it's a devDependency, but still, I'm currently fighting with modernizing the test suite because what was a valid choice 4 years ago isn't anymore, and I'm really **not** looking forward adding more dependencies.

Regarding windows, this would only impact users building it locally, not installing the lib, as the build is done by the github action. So it would only impact potential windows dev NOT using WSL. As far as I can tell, a lot of Windows devs use WSL, so it seems the right move would be to use `cp` here, instead of bringing this dependency tree:

![2024-10-26-164512_1119x651_scrot](https://github.com/user-attachments/assets/29e0ce62-ac45-4f28-8eb4-fbfa86de4f62)

Also, to be 100% honest and blunt, I couldn't care less about Windows users. This is a javascript library, not a windows desktop application, we should not limit ourselves for the abysmal percentage of windows dev that don't use WSL and could eventually contribute to this lib.

What do you think?

Reply:
I'm perfectly file just using cp. I'm just used to multi-platform development and wanted to be good citizen ;-)